authz/providers/azure/rbac/checkaccessreqhelper.go

```diff
@@ -259,8 +258,8 @@ func prepareCheckAccessRequestBody(req *authzv1beta1.SubjectAccessReviewSpec, cl
 	{
 		"Subject": {
 			"Attributes": {
-				"ObjectId": "62103f2e-051d-48cc-af47-b1ff3deec630",
-				"xms-pasrp-retrievegroupmemberships": true
+                                "ObjectId": "62103f2e-051d-48cc-af47-b1ff3deec630",
+                                "xms-pasrp-retrievegroupmemberships": false
```

Comment:
we can remove this from our schema.

Reply:
Just realized, it's in comment. Still we can remove this argument from check access call.